docs/KIS/Allgemeine Programmbedienung/.pages

```diff
@@ -0,0 +1,7 @@
+nav:
+    - Was ist das Kanzlei-Informationssystem?.md
+    - Programmstart.md
+    - 
+    - 
+    - 
+    - 
```

Comment:
Diese Zeilen lassen den Build brechen.
Hier muss ein gültiger Pfad zu einer Datei/zu einem Ordner stehen oder einmal in der Datei ist auch der Wert `...` gültig. Damit wird definiert, dass alle weiteren Dateien in dem Ornder, welche nicht explizit definiert wurden, in alphabetischer Reihenfolge angezeigt werden sollen.